docs/source/reference/datasets.rst

```diff
@@ -0,0 +1,78 @@
+.. module:: chainer.datasets
+
+.. _datasets:
+
+Dataset examples
+================
+
+The most basic :mod:`~chainer.dataset` implementation is an array.
+Both NumPy and CuPy arrays can be used directly as datasets.
+
+In many cases, though, the simple arrays are not enough to write the training procedure.
+In order to cover most of such cases, Chainer provides many built-in implementations of datasets.
+
+These built-in datasets are divided into two groups.
+One is a group of general datasets.
+Most of them are wrapper of other datasets to introduce some structures (e.g., tuple or dict) to each data point.
+The other one is a group of concrete, popular datasets.
```

Comment:
I think comma and "and" are interchangeable when we connect (coordinate) adjectives.